manager-bundle/src/Resources/skeleton/config/parameters.yml

```diff
@@ -2,13 +2,15 @@
 parameters:
     database_host: localhost
     database_port: 3306
-    database_user: ~
-    database_password: ~
-    database_name: ~
+    database_user: ''
```

Comment:
@aschempp This is required for your workaround with setting `env(DATABASE_URL)` in the parameters.yml file. The value would otherwise be `null` in a fresh installation and the `EnvVarProcessor` class would throw an exception:

https://github.com/symfony/symfony/blob/5b5b7de6609e88ad48d2b6036e1ce609aaee0388/src/Symfony/Component/DependencyInjection/EnvVarProcessor.php#L230-L232

I can check if I can work around this with a compiler pass.

Reply:
Changed in 13fb4257b8ec9b9fdd3e6e2df5191f94c41c1586.